components/VM_Arm/CMakeLists.txt

```diff
@@ -127,6 +127,18 @@ config_option(
     OFF
 )
 
+config_option(
```

Comment:
I wonder, does this have to be a config option or can we just make this feature available everywhere. We could check if the string `dtb_base_name` is set or not and then act accordingly.